### PR TITLE
Update instruction of IAP

### PR DESCRIPTION
**Pull request by blackzlq**
Update README

### CODE REVIEW

File: applications/rag/README.md [hunk 1]
```diff
@@ -105,7 +105,11 @@ gcloud container clusters get-credentials ${CLUSTER_NAME:?} --location ${CLUSTER
 1. Verify Kuberay is setup: run `kubectl get pods -n ${NAMESPACE:?}`. There should be a Ray head (and Ray worker pod on GKE Standard only) in `Running` state (prefixed by `example-cluster-kuberay-head-` and `example-cluster-kuberay-worker-workergroup-`).
 
 2. Verify Jupyterhub service is setup:
```
Comment (imreddy13) [inline applications/rag/README.md:107]: We also need to update the frontend instructions for IAP - see https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/README.md#launch-the-frontend-chat-interface

File: applications/rag/README.md [hunk 2]
```diff
@@ -105,7 +105,11 @@ gcloud container clusters get-credentials ${CLUSTER_NAME:?} --location ${CLUSTER
...
-    * Fetch the service IP: `kubectl get services proxy-public -n ${NAMESPACE:?} --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
+    * Fetch the service IP/Domain:
+      * IAP disable: `kubectl get services proxy-public -n $NAMESPACE --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
+      * IAP enabled: Read output in terminal or use commend: `kubectl get managedcertificates jupyter-managed-cert -n $NAMESPACE --output jsonpath='{.status.domainStatus[0].domain}'`
```
[the remark this answers is not in the document]
Comment (blackzlq) [reply applications/rag/README.md:110]: After the terminal apply, it will print the frontend_url

File: applications/rag/README.md [hunk 3]
```diff
@@ -91,7 +91,11 @@ gcloud container clusters get-credentials ${CLUSTER_NAME:?} --location ${CLUSTER
 1. Verify Kuberay is setup: run `kubectl get pods -n ${NAMESPACE:?}`. There should be a Ray head (and Ray worker pod on GKE Standard only) in `Running` state (prefixed by `example-cluster-kuberay-head-` and `example-cluster-kuberay-worker-workergroup-`).
 
 2. Verify Jupyterhub service is setup:
-    * Fetch the service IP: `kubectl get services proxy-public -n ${NAMESPACE:?} --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
+    * Fetch the service IP/Domain:
```
Comment (umeshkumhar) [inline applications/rag/README.md:94]: also mention to check terraform output.. we already output the frontend IP.. in both cases.

**Review by umeshkumhar — APPROVED** [review]

**Pull request merged**